aws-android-sdk-s3/src/main/java/com/amazonaws/mobileconnectors/s3/transferutility/UploadTask.java

```diff
@@ -310,8 +315,6 @@ private Boolean uploadSinglePartAndWaitForCompletion() {
                      */
                     updater.updateState(upload.id, TransferState.WAITING_FOR_NETWORK);
                     LOGGER.debug("Network Connection Interrupted: " + "Moving the TransferState to WAITING_FOR_NETWORK");
-                    ProgressEvent resetEvent = new ProgressEvent(0);
-                    resetEvent.setEventCode(ProgressEvent.RESET_EVENT_CODE);
```

Comment:
Same as above. If `eventCode` is not being used anywhere should we consider removing it altogether from the `ProgressEvent` class?